deployment/v3/mosip/ida/README.md

```diff
@@ -4,5 +4,10 @@
 ```
 ./install.sh
 ```
+* During the execution of the `install.sh` script, a prompt will appear, requesting information regarding the presence of a public domain and a valid SSL certificate on the server.
```

Comment:
During the execution of the `install.sh` script, a prompt appears requesting information regarding the presence of a public domain and a valid SSL certificate on the server.